signer/src/storage/model.rs

```diff
@@ -93,12 +93,22 @@ impl From<&crate::message::SweepTransactionInfo> for SweepTransaction {
 
 impl utxo::GetFees for Vec<SweepTransaction> {
     /// Return the total fee of all the transactions in the vector.
-    fn get_fees(&self) -> Fees {
-        let total = self.iter().map(|tx| tx.fee).sum();
-        let total_size: u32 = self.iter().map(|tx| tx.vsize).sum();
+    fn get_fees(&self) -> Result<Fees, Error> {
+        let total: u64 = self
+            .iter()
+            .map(|tx| tx.fee)
+            .try_fold(0u64, |acc, fee| acc.checked_add(fee))
+            .ok_or(Error::ArithmeticOverflow)?;
```

Comment:
Hmmm, these fees cannot add up to anything more than `u64::MAX`, even if we added up all bitcoins (measured in sats).

Reply:
Nope, but the code otherwise is _technically_ not panic-safe, which is what we've been striving for.